InOut/CMakeLists.txt

```diff
@@ -86,13 +86,9 @@ if(APPLE)
         target_link_libraries(rtauhal PRIVATE ${AUDIOUNIT_LIBRARY} ${COREFOUNDATION_LIBRARY} ${COREAUDIO_LIBRARY})
 endif()
 
-if(USE_ALSA AND LINUX)
-    if (USE_VCPKG)
-        find_package(ALSA MODULE REQUIRED)
-    else()
-        find_package(ALSA REQUIRED)
-    endif()
-
+find_package(ALSA)
```

Comment:
This will use the MODULE search instead of CONFIG then no?

Reply:
Yes, but there is no ALSA module so nothing to worry about I don't think

Reply:
https://cmake.org/cmake/help/latest/module/FindALSA.html There is the built in one for CMake

Reply:
Oh yeah. Well, I'm not sure how this change snuck in because it isn't relevant anyway, so got rid ofit.